_topic_map.yml

```diff
@@ -1488,15 +1488,13 @@ Topics:
   File: applying-autoscaling
 - Name: Creating infrastructure machine sets
   File: creating-infrastructure-machinesets
+- Name: Adding a RHEL compute machine
+  File: adding-rhel-compute
+- Name: Adding more RHEL compute machines
+  File: more-rhel-compute
```

Comment:
I would preserve the `Distros: openshift-enterprise` for these, unless you know for sure this should be displayed for all distros. Since this is RHEL-related, I'm thinking we only wanted to target `openshift-enterprise` distros. Item four [here](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc#topic-map-file-format) gives a little background.